packages/core/src/radio-group/radio.scss

```diff
@@ -1,58 +1,191 @@
+// :host {
+//   --_background: #fff;
+//   --_foreground: #fff;
```

Comment:
Do we want to keep those comments?

Reply:
This was just a work in progress, the rest was done in a separate branch :D